code/modules/mob/living/simple_animal/friendly/snek.dm

```diff
@@ -0,0 +1,22 @@
+/mob/living/simple_animal/friendly/snek
+	name = "Snake"
+	desc = "Cute snek that needs a little boop"
+	icon_state = "Snek"
+	icon_living = "snek_dead"
+	speak_emote = list("flicks tongue")
```

Comment:
This would probably look pretty weird, so I recommend something like `flicks`. Otherwise, you'll see stuff like:
`The snake flicks tongue, "Hello"`